### PR TITLE
contributors: update some project practices

### PR DESCRIPTION
**Pull request by garlick**
Some minor updates to RFC 1 and 2
* add forward references from RFC 1 to 2 and 7
* drop text about AUTHORS files we no longer use
* suggest the `Problem:` format for commit message bodies

### CODE REVIEW

File: spec_1.rst [hunk 1]
```diff
@@ -74,15 +81,15 @@ Preliminaries
 Licensing and Ownership
 ~~~~~~~~~~~~~~~~~~~~~~~
 
--  The project SHALL use a share-alike license, such as the GPLv3 or a variant thereof (LGPL, AGPL), or the MPLv2.
+-  The project SHALL use a share-alike license, such as the GPLv3 or a variant thereof (LGPL, AGPL), or the MPLv2 for reasons outlined in RFC 2.
```
Comment (SteVwonder) [inline spec_1.rst:84]: Do we want to include the link inline as well?

```suggestion
-  The project SHALL use a share-alike license, such as the GPLv3 or a variant thereof (LGPL, AGPL), or the MPLv2 for reasons outlined in :doc:`RFC 2 <spec_2>`.
```

File: spec_1.rst [hunk 2]
```diff
@@ -26,6 +26,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to
 be interpreted as described in `RFC 2119 <http://tools.ietf.org/html/rfc2119>`__.
 
+Related Standards
+-----------------
+
+-  :doc:`2/Flux Licensing and Collaboration Guidelines <spec_2>`
+
+-  :doc:`7/Flux Coding Style Guide <spec_2>`
```
Comment (SteVwonder) [inline spec_1.rst:34]:
```suggestion
-  :doc:`7/Flux Coding Style Guide <spec_7>`
```

**Review by SteVwonder — COMMENTED** [review]
Thanks @garlick!  Generally LGTM!  One copy-paste bug and a question:

File: spec_1.rst (hunk 2 above)
Comment (garlick) [reply spec_1.rst:34]: DOH

**Comment by garlick** [comment]
Thanks for catching those things @SteVwonder !  I force pushed an update.

**Review by grondo — APPROVED** [review]
LGTM too. Thanks!

**Review by SteVwonder — APPROVED** [review]
LGTM! Thanks @garlick!

**Pull request merged**